test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -532,11 +533,23 @@ public void TimezoneConnectionParam()
 
         #endregion Timezone
 
-        [Test, Explicit("Doesn't work on Windows, sometimes not available Linux (e.g. when PG is in docker)")]
+        [Test]
```

Comment:
I'd wait and see whether we run into problems with this.
I simply don't believe the docker argument (you obviously need some configuration if you test a server running in a container from outside the container but you can always mount your socket as a file and access it) and I'm also not aware of any test configuration where anyone even does use docker containers for testing.

Reply:
This just failed on my machine because... the tests authenticate as npgsql_tests by default, but my Unix username is roji. I'm also not sure it's correct to assume that peer auth will always be enabled by default in PostgreSQL, across all distributions etc.

As a general rule, it's important for all tests to pass out-of-the-box - otherwise contributors aren't sure if they broke something or not... Do you have an objection to making this Explicit again?

Reply:
As I know, psql assumes that it's enabled by default if another option isn't specified. So we should have the same behaviour.

Reply:
> psql assumes that it's enabled by default if another option isn't specified

I'm pretty sure different PG distributions may decide on different settings - I don't like to depend on this kind of thing. And in any case there's the username mismatch issue I just ran into.

Reply:
>> I'd wait and see whether we run into problems with this.

> This just failed on my machine because [...] Do you have an objection to making this Explicit again?

The natural consequence of what I wrote above is, that I don't have objections.

Reply:
#2955 - ended up doing the best of both worlds. Instead of [Explicit], failures are just ignored unless in CI (just like when a socket isn't found).